app/services/requirements/checker.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Requirements
+  class Checker
```

Comment:
I'm curious about the relationship between this class, DocumentChecker and their responsibilities. Is this class intended to cover the entire document from a user perspective and the DocumentChecker and ImageChecker are mapped to the models of `Document` and `Image` respectively or is it different?

Reply:
We discussed this and came up with a plan to split the DocumentChecker into separate classes, freeing up the name 'DocumentChecker' to be the thing that runs all the checks.